### DISCUSSION TITLE
Store api

### DISCUSSION THREAD

**Discussion by marijang** [woocommerce/woocommerce#42963] [Checkout Flow]
I m planing to use store api in mine gutenberg blocks?
My question is can i get cart object on php side the same as with rest service?

**Comment by marijang** [comment]
How to add demo data in editor blocks?
Is there rest api?

**Comment by opr** [comment]
Hi there @marijang ! thanks for opening a discussion here, but we actually moved over to the [WooCommerce Monorepo](https://github.com/woocommerce/woocommerce/) so I'll transfer your discussion there.

To answer your first question:

> I m planing to use store api in mine gutenberg blocks?
> My question is can i get cart object on php side the same as with rest service?

It's a little unclear what _exactly_ your'e asking here, but during a Store API request the cart should be available and accessible through `WC()->cart`.

For your section question:

> How to add demo data in editor blocks?
> Is there rest api?

This is again unclear, are you talking about the Cart/Checkout blocks? What demo data would you like to add that isn't already there?
